modules/schema/pages/notice-information.adoc

```diff
@@ -178,9 +181,9 @@ This information does not exist for notices of "_Planning_" form type.
 [[dispatchDateSection]]
 == Notice Dispatch Date
 
-The *+++date and time+++* of the notice receipt by OP. This date will be
+The *+++date and time+++* of the notice submission by the Buyer. This date will be
 added before its submission to OP and its trustworthiness evaluated
-based on the actual receipt time and a range of acceptable variationsfootnote:[To be defined by Business].
+based on the actual receipt time and a range of acceptable variationsfootnote:[CVS checks the value against the current date. The rule is currently permissive, allowing for "Notice Dispatch Date (BT-05), or Notice Dispatch Date eSender (BT-803) if provided, to be 1 day before or after the current date." Rule will get tightened to 24 hours before the current date as of SDK 2.].
```

Comment:
I think we should also explain when it is useful to provide "Notice Dispatch Date eSender".
From what I know, it's when the processing by the eSender introduces a significant delay after the submission by the buyer, to avoid falling outside of the acceptable range.